.gitignore

```diff
@@ -4,3 +4,4 @@ deb_dist
 dist
 *.pyc
 rosdistro.egg-info
+.coverage
```

Comment:
I prefer the .gitignore lines to be lexicographically sorted (and prefer directories to have trailing slashes) but since neither of those conventions are adhered to in the current contents moving .coverage to the top is optional as I'll have to make a separate PR bringing us into conformance generally.

Reply:
No reason to make things worse. Done in e03cc3d.